requirements.txt

```
@@ -40,4 +40,4 @@ s3fs
 gcsfs
 # adlfs
 
-trl @ git+https://github.com/huggingface/trl.git@main
+trl==0.7.9
```

```suggestion
trl>=0.7.9
```
to make it a bit more future proof